docs/pages/reference/networking.mdx

```diff
@@ -232,12 +243,8 @@ traffic:
 | - | - | - |
 | 3022 | SSH Service | Incoming SSH connections.|
 | 3026 | Kubernetes Service | HTTPS traffic to a Kubernetes API server.| 
-| 3036 | Database Service | Traffic to MySQL databases.|
-| 5432 | Database Service | Traffic to Postgres databases.|
-| 27017 | Database Service | Traffic to MongoDB instances.|
-| 6379 | Database Service | Traffic to Redis instances.|
 | 3028 | Windows Desktop Service | Teleport Desktop Protocol traffic from Teleport clients.|
 
-Applications registered with the Teleport Application Service can only be
-accessed via the Teleport Proxy Service, not directly via the Application
+Applications, and Databases registered with their respective Teleport Service
+can only be accessed via the Teleport Proxy Service, not directly via their
 Service.
```

Comment:
```suggestion
You can only access enrolled applications and databases through the Teleport Proxy Service.
You can't access them directly using the Teleport Application Service or Teleport Database Service.
```
I'm not sure I captured your intended meaning because the sentence is a little bit awkward (maybe trying to convey too much?).